walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,79 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.ArrayList;
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+    ArrayList<TimeRange> busyTimeRanges = new ArrayList();
+    ArrayList<TimeRange> freeTimeRanges = new ArrayList();
+
+    //If there are no attendees, whole day is free
```

Comment:
nit. comments should start with a space, so this should instead be "// If there are no..."